usecases/auth/authorization/rbac/manager.go

```diff
@@ -175,11 +170,70 @@ func (m *manager) Authorize(principal *models.Principal, verb string, resources
 			return err
```

Comment:
wouldn't we miss any errors here in the log?

Reply:
sorry, don't get it isn't the error logged already ?